Automattic/jetpack
# Subscription form: Match placeholder color to the theme

This updates the color of the placeholders in the Subscription Form block so that they match the theme.

Before:
<img width="738" alt="Screenshot 2022-01-07 at 11 08 02" src="https://user-images.githubusercontent.com/275961/148535724-1116f87d-e8c4-481e-9378-dfb05e8acf8c.png">

After:
<img width="743" alt="Screenshot 2022-01-07 at 10 56 49" src="https://user-images.githubusercontent.com/275961/148535727-df403efd-3c4a-450a-a9d2-668a0be59d12.png">

#### Changes proposed in this Pull Request:
* The default color for placeholders is different in different browsers. This change ensures that the block applies the text color from the theme to the placeholder.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Spin up a site with Jetpack installed using a non-local domain
* Enable the subscriptions module in settings
* Add a subscription form block to a post
* Check that the placeholder text color matches the color of the theme.
* Switch themes and check that the placeholder color changes to match the theme
